recipe/activate-clang.sh

```diff
@@ -153,6 +153,14 @@ if [ "@CONDA_BUILD_CROSS_COMPILATION@" = "1" ]; then
   echo "cpu = '@UNAME_MACHINE@'" >> ${CONDA_PREFIX}/meson_cross_file.txt
   echo "cpu_family = '@MESON_CPU_FAMILY@'" >> ${CONDA_PREFIX}/meson_cross_file.txt
   echo "endian = 'little'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  # specify path to correct binaries/metadata the meson will not auto-discover out of caution
+  # binaries from build env
+  echo "[binaries]" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  echo "cmake = '${CONDA_PREFIX}/bin/cmake'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  echo "pkgconfig = '${CONDA_PREFIX}/bin/pkg-config'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  # metadata from host env
+  echo "[properties]" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  echo "pkg_config_libdir = '${PREFIX}/lib'" >> ${CONDA_PREFIX}/meson_cross_file.txt
```

Comment:
Do this only if ${PREFIX} is defined.

Reply:
I pushed a commit to https://github.com/conda-forge/scipy-feedstock/pull/205 to try what happens without. I was following the upstream cross-compilation recommendations, but it's _possible_ that our infra doesn't need this bit of info.

Reply:
Commonly a cross-compile toolchain provides a custom pkg-config that actually just sets that variable and then executes the "real" pkg-config. The ultimate goal is to instruct pkg-config what configuration it can use, via several possible methods:
- compile-time settings
- environment variable overrides
- pkgconf-personality files: https://manpages.debian.org/testing/pkgconf/pkgconf-personality.5.en.html

If you compiled pkg-config for native, you can use the *same* binary for cross as well, and all you need to do is set environment variables rather than rebuilding the whole thing. But ideally you can do something like `<triplet>-pkg-config` or `/path/to/custom/toolchain/bin/pkg-config` and that will set things up automatically.